src/stores/business.ts

```diff
@@ -213,6 +214,270 @@ export const useBcrosBusiness = defineStore('bcros/business', () => {
     return !!requestedFiling
   }
 
+  /** Check if the specified action is allowed, else False */
+  const isAllowed = (action: AllowableActionE): boolean => {
+    // const isBusiness = !!sessionStorage.getItem('BUSINESS_ID') // ie, not a temporary business
+
+    // TO-DO: the above line is commented out because we do not have 'BUSINESS_ID' in the sessionStorage
+    // For now, we check if the currentBusiness exists in the business store.
+    const isBusiness = !!useBcrosBusiness().currentBusiness.identifier
```

Comment:
If we are keeping it here, I think you dont have to use Business store, as you are inside, am not sure what are consequences of that.

Reply:
Good catch! I will change that.  Another update I need to make is, only one todo item can be expanded each time.